sklearn/tests/test_base.py

```diff
@@ -436,6 +436,7 @@ def __init__(self, attribute_pickled=5):
 
     def __getstate__(self):
         data = self.__dict__.copy()
+        data["_sklearn_version"] = sklearn.__version__
```

Comment:
Thanks a lot for the details @HolgerPeters!

@zxcvbnius I think your change is fine. Add a comment on the previous line explaining that the line you added avoids a DeprecationWarning about unpickling an estimator from a different version.